spec.go

```diff
@@ -1,15 +1,14 @@
 package flanneltpr
 
 import (
-	"github.com/giantswarm/flanneltpr/host"
-	"github.com/giantswarm/flanneltpr/network"
+	"github.com/giantswarm/flanneltpr/bridge"
+	"github.com/giantswarm/flanneltpr/cluster"
+	"github.com/giantswarm/flanneltpr/flannel"
 )
 
 type Spec struct {
-	Host host.Host `json:"host" yaml:"host"`
-	// Namespace is the namespace of the guest cluster to watch for. E.g. Flannel
-	// bridges should not be removed until there is any workload running within
-	// the guest cluster's namespace.
-	Namespace string          `json:"namespace" yaml:"namespace"`
-	Network   network.Network `json:"network" yaml:"network"`
+	Cluster cluster.Cluster `json:"cluster" yaml:"cluster"`
+
+	Bridge  bridge.Bridge   `json:"bridge" yaml:"bridge"`
+	Flannel flannel.Flannel `json:"flannel" yaml:"flannel"`
```

Comment:
Now in the TPO spec we have only `Cluster` config, `Bridge` config, and `Flannel` config. They are logically separate. The only issue is that `flanneld` needs the bridge name, but it seemed obvious to me that the bridge's name belongs to `Bridge`.